src/Apps/Settings/Routes/Insights/InsightsRoute.tsx

```diff
@@ -0,0 +1,22 @@
+import { Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { InsightsRoute_me } from "__generated__/InsightsRoute_me.graphql"
+
+interface InsightsRouteProps {
+  me: InsightsRoute_me
+}
+
+const InsightsRoute: React.FC<InsightsRouteProps> = ({ me }) => {
+  return <Text>Sup</Text>
+}
+
+export const InsightsRouteRefetchContainer = createFragmentContainer(
```

Comment:
👀 This is named `InsightsRouteRefetchContainer` but you have not used `RefetchContainer`.

Reply:
nice catch!! I wanted to name it FragmentContainer 😂